### DISCUSSION TITLE
HelloGitHub Badge

### DISCUSSION THREAD

**Discussion by 521xueweihan** [Devolutions/UniGetUI#2703] [General]
We're thrilled to share that [your project](https://hellogithub.com/en/repository/46bec642537f449a857215e39a1d64ae) has caught the attention of the HelloGitHub community and has been recognized for its merit. Your work is truly inspiring, and we'd like to invite you to participate in our [HelloGitHub Badge Program](https://hellogithub.com/en/repository/46bec642537f449a857215e39a1d64ae/embed).

<img width="775" alt="image" src="https://github.com/user-attachments/assets/688a22c0-7489-410e-ba10-70c9aaeae281">

By joining, you'll enjoy a few benefits that we believe will complement your project's journey:

1. **Community Acknowledgement**: The badge serves as a mark of distinction, indicating that your project has met the community's criteria for recommendation.
2. **Visibility Boost**: Wearing the badge can help increase the visibility of your project, potentially drawing the attention of new users and contributors.
3. **Easier Engagement**: The badge acts as a quick reference for users to understand your project, facilitating interaction through likes, comments, and bookmarks.
4. **Feedback Opportunities**: It opens the door for valuable feedback from a diverse user base, which can be instrumental in your project's growth and refinement.
5. **Notable Recognition**: Verified participants will receive a special identifier, highlighting their contributions within the community.

We invite you to consider joining the [HelloGitHub Badge Program](https://hellogithub.com/en/repository/46bec642537f449a857215e39a1d64ae/embed) as a way to further engage with the community and enhance your project's presence. It's completely up to you, and we respect your decision either way.

Thank you for being part of this journey with us.

Warm regards,
The HelloGitHub Team

---

HelloGitHub since launch in 2016 as a monthly newsletter, we've evolved into a thriving community that celebrates and enhances the reach of open-source projects. Today, our community is home to over 20,000 enthusiastic members, and our [open-source project](https://github.com/521xueweihan/HelloGitHub) have garnered a stellar 90,000 GitHub stars.

**Comment by Lockszmith-GH** [comment]
I'm not a maintainer here, just a semi-active fan of the project.

Never heard of your project before, and it does look very nice and pleasant on the eyes.

The open source project's license and EULA for the main site don't seem to match.

With your project's focus on GitHub, open-source and Free Software should be the guidline for your project as well, but that does not seem to be the case.
The agreement feels like it's run by a data-harvesting project and not a colaborative create-commons type project.

Overall, the agreement places significant control in the hands of HelloGitHub, which is contrary to the principles of free software that emphasize user freedom, transparency, and autonomy. 

Could you explain the dissonance between these two aspects of your community?

**Reply by 521xueweihan** [reply]
@Lockszmith-GH Thank you for your reply. Here is my explanation for your question:

1. The EULA of the main site must comply with the laws of my country. The content review must be reviewed by the administrator, so I need to review the content in the background before releasing it to ensure the compliance of the content.

2. The front-end source code of the website has been open sourced [address](https://github.com/HelloGitHub-Team/geese). Since I am the only one developing it now, the development progress is slow. It took me two months to develop the English version.

Sometimes, a road needs to be paved by one person before there will be followers. I don’t know if my answer has solved your doubts. Welcome to discuss.
